foreman_google.gemspec

```diff
@@ -16,7 +16,13 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib,locale,webpack}/**/*'] + ['LICENSE', 'Rakefile', 'README.md', 'package.json']
   s.test_files = Dir['test/**/*'] + Dir['webpack/**/__tests__/*.js']
 
-  s.add_dependency 'google-apis-compute_v1', '~> 0.14'
-  s.add_dependency 'google-cloud-compute', '~> 0.2'
+  # Pin Google versions to avoid breaking changes
+  # Never versions with google-protobuf > 3.25.4
+  # are failing with `undefined method 'build'` error
+  s.add_dependency 'google-apis-compute_v1', '~> 0.14.0'
```

Comment:
This is incorrect; it should match the version that we have in the packaging repository.
https://github.com/theforeman/foreman-packaging/tree/rpm/develop/packages/foreman/rubygem-google-apis-compute_v1